src/main/java/csci432/model/Blob.java

```diff
@@ -0,0 +1,160 @@
+package csci432.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.awt.*;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Blob class for keeping track of moving items
+ **/
+public class Blob {
+    @JsonProperty("locations")
+    private Map<Integer, Point> locations;
```

Comment:
Add a comment that shows the significance of the Integer as the key and the Point as the data. Where is this value supposed to come from? Adding a doc comment for the constructor might fix this too.